server/container_create.go

```diff
@@ -350,7 +350,7 @@ func (s *Server) createSandboxContainer(ctx context.Context, containerID string,
 	attempt := metadata.GetAttempt()
 	containerInfo, err := s.storage.CreateContainer(s.imageContext,
 		sb.name, sb.id,
-		image, "",
+		image, image,
```

Comment:
Sure, I think we could sort them using `containers/image/docker/reference.ParseIDOrReference` in `CreateContainer()`.